src/custom/state/orders/actions.ts

```diff
@@ -26,9 +26,24 @@ export interface BaseOrder extends Omit<OrderCreation, 'signingScheme'> {
   isUnfillable?: boolean // whether the order is out of the market, due to price movements since placement
 }
 
+/**
+ * The API provides some additional information
+ */
+type OrderInfoApi = Pick<
+  OrderMetaData,
+  | 'creationDate'
+  | 'availableBalance'
+  | 'executedBuyAmount'
+  | 'executedSellAmount'
+  | 'executedSellAmountBeforeFees'
+  | 'executedFeeAmount'
+  | 'invalidated'
+>
```

Comment:
That works.
You also also store the whole OrderMetadata obj and skip this intermediate type.

Any harm in doing that?

Reply:
would be alright too, is just for not including the whole bunch of stuff we don't need

Reply:
is `Omit<>` not a better choice here? or are there that many keys that it's less to Pick over Omit?

Reply:
There's a lot of keys, also easier to read the Pick than the Omit imo, since is explicit what u "pick"